File: c/1-sequence-struct/sum/main.c
// Write a program that takes two integers and prints the sum.

#include <stdio.h>
#include <stdlib.h>

int main()
{
    int n1, n2, sum;

    printf("Enter the first number: ");
    scanf("%d", &n1);
    printf("Enter the second number: ");
    scanf("%d", &n2);

    sum = n1 + n2;

    printf("\nThe sum of the numbers entered is %d.\n", sum);
    return 0;
}

File: java/1-sequence-struct/sum/src/Main.java
// Write a program that takes two integers and prints the sum.

import java.util.Scanner;

public class Main {
    public static void main(String[] args) {

        // Instantiating object in java for data entry
        Scanner sc = new Scanner(System.in);

        int n1, n2, sum;

        System.out.print("Enter the first number: ");
        n1 = sc.nextInt();
        System.out.print("Enter the second number: ");
        n2 = sc.nextInt();

        sum = n1 + n2;
        System.out.println("\nThe sum of the numbers entered is " + sum);
    }
}
